index.js

```diff
@@ -379,7 +379,14 @@ function dist(ax, ay, bx, by) {
 }
 
 function orient(px, py, qx, qy, rx, ry) {
-    return (qy - py) * (rx - qx) - (qx - px) * (ry - qy) < 0;
+    const d = cross(px, py, qx, qy, rx, ry);
+    if (Math.abs(d) > 1e-6) return d < 0;
```

Comment:
I wonder how much we could lower the `1e-6`? That may be too high for the purpose.

Reply:
In all the examples I had either large numbers, or very small ones (for triangles that were definitely problematic). In those examples it looked like the small value was ~1e-11 or less, but I don't know when the precision stops being a problem.

(In any case the cost for such small triangles is just +2 area computations.)